apps/dashboard/src/app/event/components/pools-box.tsx

```diff
@@ -1,38 +1,34 @@
 import { notifyFail } from "@/lib/notifications"
-import type { AttendancePool } from "@dotkomonline/types"
+import type { Attendance, AttendancePool } from "@dotkomonline/types"
 import { Box, Button, Card, Flex, Grid, Text, Title } from "@mantine/core"
 import type { FC } from "react"
 import { useDeletePoolMutation } from "../mutations"
 import { openEditPoolModal } from "./edit-pool-modal"
 
-interface PoolsBoxProps {
-  pools: AttendancePool[]
-  attendanceId: string
-}
-
 interface NormalPoolBoxProps {
   pool: AttendancePool
-  attendanceId: string
+  attendance: Attendance
   deleteGroup: (id: string, numAttendees: number) => void
 }
 
-const AttendancePoolCard: FC<NormalPoolBoxProps> = ({ pool, attendanceId, deleteGroup }) => {
+const AttendancePoolCard: FC<NormalPoolBoxProps> = ({ pool, attendance, deleteGroup }) => {
+  const attendeeCount = attendance.attendees.filter((attendee) => attendee.attendancePoolId === pool.id).length
   return (
     <Card shadow="sm" padding="lg" radius="md" withBorder key={pool.id} mt={16}>
       <Flex justify="space-between" direction="column" gap="lg">
         <Box>
           <Title order={2}>{pool.title}</Title>
           <Text>
             {pool.capacity > 0
-              ? `${Math.min(pool.numAttendees, pool.capacity)} / ${pool.capacity} påmeldte`
+              ? `${Math.min(attendeeCount, pool.capacity)} / ${pool.capacity} påmeldte`
```

Comment:
This should show how many attendees are reserved, and not max it out on capacity, in case there actually are more reserved attendees than capacity

```suggestion
  const reservedAttendeeCount = attendance.attendees.filter((attendee) => attendee.attendancePoolId === pool.id && attendee.reserved).length
  return (
    <Card shadow="sm" padding="lg" radius="md" withBorder key={pool.id} mt={16}>
      <Flex justify="space-between" direction="column" gap="lg">
        <Box>
          <Title order={2}>{pool.title}</Title>
          <Text>
            {pool.capacity > 0
              ? `${reservedAttendeeCount} / ${pool.capacity} påmeldte`
```